src/components/SpiceList.js

```diff
@@ -8,11 +8,33 @@ class SpiceList extends React.Component {
   }
 
   renderSpices() {
-    return this.props.spices.map(spice => (
+    let filteredSpice = this.props.spices.filter(spice => {
+      return spice.notes.toLowerCase().includes(this.state.search.toLowerCase())
+    })
+
+    if (this.state.fourStarOnly){
+      filteredSpice = filteredSpice.filter(spice => spice.rating >= 4)
+    }
+
+    return filteredSpice.map(spice => (
```

Comment:
Nice work getting these multiple filters set up! If you want to see a different approach using some method chaining, here's another way you could achieve the same filtering:

```js
let filteredSpices = this.props.spices
      .filter(spice => this.state.fourStarOnly ? spice.rating >= 4 : true)
      .filter(spice => spice.notes.toLowerCase().includes(this.state.search.toLowerCase()))
```